CHANGELOG.md

```diff
@@ -1031,6 +1031,8 @@
     demangle-name-section = true
     # Should we emit the DWARF debug info custom sections?
     dwarf-debug-info = false
+    # Should we omit the default import path?
+    omit-default-module-path = false
```

Comment:
I don't think the old changelog should be edited.

Reply:
Oops 🙈 Thanks for catching this one 🙏 Added a revert commit for CHANGELOG.